packages/styles/src/lib/Theme/components/MuiChip.ts

```diff
@@ -0,0 +1,96 @@
+import { Components } from '@mui/material';
+import { palette as themePalette } from '../palette';
+import { themeTypography } from '../typography';
+import '../fonts.css';
+
+// This adds 50% transparency to hexadecimal colors
+const addTransparency = (color?: string): string => `${color}80`;
```

Comment:
can we make this function more customizable, passing the transparency and make it as a util inside of styles package?

Reply:
It's a little bit confusing to read but sure, no problem